sway-lib-std/src/array_conversions/b256.sw

```diff
@@ -0,0 +1,201 @@
+library;
+
+use ::array_conversions::u64::*;
+use ::assert::assert;
```

Comment:
Instead of importing to the entire file you can just import at the test level.

Reply:
You mean assert? But im using it in the implementations

Reply:
Ah then disregard